### PR TITLE
Fix bad selection of monitor(s) from k8s

### PR DESCRIPTION
**Pull request by JrCs**

### CODE REVIEW

File: ceph-releases/jewel/ubuntu/14.04/daemon/config.k8s.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+
```
Comment (leseb) [inline ceph-releases/jewel/ubuntu/14.04/daemon/config.k8s.sh]: ?

Comment (JrCs) [reply ceph-releases/jewel/ubuntu/14.04/daemon/config.k8s.sh]: It is just for the readiness

Comment (JrCs) [reply ceph-releases/jewel/ubuntu/14.04/daemon/config.k8s.sh]: Do you want that i remove the empty line ?

**Comment by leseb** [comment]
jenkins test this please

**Comment by osixia** [comment]
Why did you remove the namespace ?!

**Comment by JrCs** [comment]
Namespace is not necessary because the pods are start on the right namespace and inside the containers of the pods the default namespace is the namespace of the pods

**Comment by osixia** [comment]
Thanks,
but should **check_zombie_mons.py** use **K8S_MON_SELECTOR** variable too ?

**Comment by JrCs** [comment]
@ghost you're right.
Add a new PR #860.

**Pull request merged**